src/config.c

```diff
@@ -3031,9 +3029,8 @@ standardConfig createModuleStandardConfig(const char *config_name, int flags, vo
 }
 
 /* Create and add a skeleton standardConfig for a module config to the configs array */
-void addModuleConfig(const char* module_name, const char* name, int flags, void *privdata) {
-    sds config_name = sdsnew(module_name);
-    config_name = sdscat(sdscat(config_name, "."), name);
+void addModuleConfig(const char *module_name, const char *name, int flags, void *privdata) {
+    sds config_name = sdscatfmt("%s.%s", module_name, name);
```

Comment:
Sorry, my suggestion is missing `sdsempty()`.
Should be `sds config_name = sdscatfmt(sdsempty(), "%s.%s", module_name, name);`

Reply:
yeah, caught this when running with asan. no problem